test.disabled/ejabberd_tests/tests/mod_blocking_SUITE.erl

```diff
@@ -286,6 +287,38 @@ blocking_propagates_to_resources(Config) ->
             message_is_not_delivered(User2, [User1b], <<"miau!">>)
         end).
 
+send_iq_with_blocking_ns(Config) ->
+    %% Will send iq with blocking namespace with type result and error
+    escalus:story(Config, [{alice, 1}, {bob, 1}], fun(Alice, Bob) ->
+        QueryWithPrivacyNS = escalus_stanza:query_el(?NS_BLOCKING, []),
+        BobJid = escalus_utils:get_short_jid(Bob),
+        IQError = escalus_stanza:iq(BobJid,
```

Comment:
Lines 295-306 may be extracted into parametrised helper in order to reduce code duplication. The same applies to `privacy_SUITE`.